src/messages.ts

```diff
@@ -0,0 +1,73 @@
+import {BeaconBlockBody, BeaconBlockHeader, BeaconBlockRoot} from "./types";
+
+export interface RpcBody {
+  method_id: number;
```

Comment:
consider using camel case for these

Reply:
I was thining runtime issues but obviously not so good point!